README.md

```diff
@@ -110,6 +111,10 @@ The following options are also available:
   `#{ENV['HOME']}/.chef/taste-tester-ref.txt`
 * checksum_dir - The checksum directory to put in knife.conf for users. Default:
   `#{ENV['HOME']}/.chef/checksums`
+* bundle - use a single tar.gz file for transporting cookbooks, roles and
```

Comment:
I would add some verbiage here about how this is intended for parallel testing if the same set of sources, but is *not* intended for use for general testing.

Reply:
I've added that this is experimental. I think that it is too early to make judgements about whether this is appropriate for general testing. I'll address that in the other comment.